apps/site/package.json

```diff
@@ -103,16 +104,16 @@
     "@types/react": "^18.3.5",
     "@types/react-dom": "^18.3.0",
     "@types/semver": "~7.5.8",
-    "eslint": "^9.9.1",
-    "eslint-config-next": "~14.2.3",
-    "eslint-config-prettier": "9.1.0",
-    "eslint-import-resolver-typescript": "^3.6.3",
-    "eslint-plugin-import-x": "^4.2.1",
-    "eslint-plugin-mdx": "3.1.5",
-    "eslint-plugin-no-relative-import-paths": "^1.5.3",
-    "eslint-plugin-react": "^7.35.2",
-    "eslint-plugin-react-hooks": "5.1.0-rc-4c58fce7-20240904",
-    "eslint-plugin-storybook": "0.9.0--canary.156.da7873a.0",
+    "eslint": "~9.9.1",
+    "eslint-config-next": "~14.2.6",
+    "eslint-config-prettier": "~9.1.0",
```

Comment:
From #7032, `eslint-config-prettier` can be removed since you're not using any starter configs that enable formatting rules:

```suggestion
```